proto/src/determined/api/v1/api.proto

```diff
@@ -146,17 +146,6 @@ service Determined {
       tags: "Users"
     };
   }
-  // Post a user's settings for website
-  rpc OverwriteUserSetting(OverwriteUserSettingRequest)
-      returns (OverwriteUserSettingResponse) {
-    option (google.api.http) = {
-      put: "/api/v1/users/setting",
-      body: "*"
-    };
-    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
-      tags: "Users"
```

Comment:
could this be considered breaking a public-facing api. if so could you add the label to the pr, please? 
sometimes we deprecate and delete later, it'd depend on the API.